buildSrc/src/main/groovy/io.deephaven.javadoc-conventions.gradle

```diff
@@ -0,0 +1,18 @@
+// Neither plain withType(Javadoc) {..} nor withType(Javadoc).all {..} will work here, somehow
+// this breaks sourcesets in some projects.
+tasks.withType(Javadoc).configureEach {
+    javadocTool = javaToolchains.javadocToolFor{javadocSpec ->
+        // Javadoc version >=11 is needed for search
+        // Javadoc version >12 is needed to avoid javadoc bugs in linking with modules
+        javadocSpec.languageVersion = JavaLanguageVersion.of(17)
+    }
+
+    options.encoding = 'UTF-8'
+
+    options.links = ['https://docs.oracle.com/en/java/javase/11/docs/api/']
```

Comment:
Why 11?

Reply:
I'm guessing b/c that's the language level we target.

Reply:
Yes, just centralized the javadoc configuration to one place, so everything is built the same way, rather than 3-ish different setups.